components/mitsubishi_itp/mitp_packet-get_response.cpp

```diff
@@ -115,5 +125,11 @@ std::string ErrorStateGetResponsePacket::get_short_code() const {
   return {upper_alphabet[(error_code & 0xE0) >> 5], lower_alphabet[low_bits]};
 }
 
+// StatusGetResponsePacket functions
+float StatusGetResponsePacket::get_lifetime_kwh() const {
+  uint16_t raw_value = pkt_.get_payload_byte(PLINDEX_LIFETIME_KWH) << 8 | pkt_.get_payload_byte(PLINDEX_LIFETIME_KWH + 1);
+  return (raw_value / 10.0f);
```

Comment:
Does this get reset with power outages?
Also, probably the right call to convert to kWh instead of *hectawatt hours*?!

Reply:
Honestly, no idea. None of my systems seem to support it, or it's very unclear as to what it does (is it system-wide? is it for the indoor unit only? does a branch box affect it?).

I don't _think_ it resets with power outages, but I don't have enough info to say either way. The statistic itself is marked as `TOTAL_INCREASING`, so any resets (or overflows) shouldn't mess up HA data.